rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_field_mapping.json

```diff
@@ -65,10 +65,6 @@
         ],
         "default":"open",
         "description":"Whether to expand wildcard expression to concrete indices that are open, closed or both."
-      },
-      "local":{
-        "type":"boolean",
-        "description":"Return local information, do not retrieve the state from master node (default: false)"
```

Comment:
This param for the `GetFieldMapping` API was deprecated in https://github.com/elastic/elasticsearch/pull/55014 and I think https://github.com/elastic/elasticsearch/pull/57265 aimed to propagate that deprecation to the REST API spec, but it changed `get_mapping.json` instead of `get_field_mapping.json`. https://github.com/elastic/elasticsearch/pull/55100 removed the `local` param for the _field_ mapping API but the regular mapping API still supports the `local` param. That's why I'm deleting the param here and deprecating it in `get_mapping.json` below.

Reply:
Could you pull this out to a separate PR?